src/styles/blocks/header.scss

```diff
@@ -0,0 +1,14 @@
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
```

Comment:
Box-shadow for header is absent. Pay attention to the design
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/aadf0929-9473-456f-8950-6723e340d4b7)
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/5861109d-58af-4f7c-b284-c150a8186935)
![image](https://github.com/mate-academy/layout_catalog/assets/96409563/d8eaebf4-7b27-4df5-92b3-01020bc04f08)